src/client/js/create.js

```diff
@@ -0,0 +1,346 @@
+selectedDateElementEnd.textContent = formatDate(date);
+selectedDateElementEnd.dataset.value = selectedDate;
+
+populateDates();
+populateDatesEnd();
+
+
+// 시작일 - 데이트 피커 버튼 클릭 시 토글
+function toggleStartDate(event) {
+	if (!checkEventPathForClass(event.path, "dates" )){
+		datesElement.classList.toggle("active");
+	}
+}
+// 종료일 - 데이트 피커 버튼 클릭 시 토글
+function toggleEndDate(event) {
+	if (!checkEventPathForClass(event.path, "dates" )){
+		datesElementEnd.classList.toggle("active");
+	}
+}
+
+// 시작일 - 다음 월로 이동
+function goToNextMonth(event){
+	month++;
+	if (month > 11){
+	month = 0;
+	year++;
+	}
+	monthElement.textContent = months[month] + " " + year;
+	populateDates();
+}
+
+// 시작일 - 이전 월로 이동
+function goToPrevMonth(event){
+	month--;
+	if (month < 0){
+	month = 11;
+	year--;
+	}
+	monthElement.textContent = months[month] + " " + year;
+	populateDates();
+}
+
+// 종료일 - 다음 월로 이동
+function goToNextMonthEnd(event){
+	month++;
+	if (month > 11){
+	month = 0;
+	year++;
+	}
+	monthElementEnd.textContent = months[month] + " " + year;
+	populateDatesEnd();
+}
+
+// 종료일 - 이전 월로 이동
+function goToPrevMonthEnd(event){
+	month--;
+	if (month < 0){
+	month = 11;
+	year--;
+	}
+	monthElementEnd.textContent = months[month] + " " + year;
+	populateDatesEnd();
+}
+
+nextMonthElement.addEventListener("click", goToNextMonth);
+prevMonthElement.addEventListener("click", goToPrevMonth);
+nextMonthElementEnd.addEventListener("click", goToNextMonthEnd);
+prevMonthElementEnd.addEventListener("click", goToPrevMonthEnd);
+datePickerElement.addEventListener("click", toggleStartDate);
+datePickerElementEnd.addEventListener("click", toggleEndDate);
+
+// 시작일 - 월별 일자 생성
+function populateDates(event) {
+	daysElement.innerHTML = "";
+	let amountDays = 31;
+
+	if (month == 1){
+			amountDays = 28;
+	} 
+	if (month == 3){
+		amountDays = 30;
+	}
+	if (month == 5){
+		amountDays = 30;
+	}
+	if (month == 8){
+		amountDays = 30;
+	}
+	if (month == 10){
+		amountDays = 30;
+	}
+
+    for(let i = 0; i < amountDays; i++) {
+        const dayElement = document.createElement("div");
+        dayElement.classList.add("day");
+        dayElement.textContent = i + 1;
+
+        if (selectedDay == (i + 1) && selectedYear == year && selectedMonth == month){
+            dayElement.classList.add("selected");
+        }
+
+        dayElement.addEventListener("click", function() {
+            selectedDate = new Date(year + "-" + (month + 1) + "-" + (i + 1));
+            selectedDay = (i + 1);
+            selectedMonth = month;
+            selectedYear = year;
+            
+            selectedDateElement.textContent = formatDate(selectedDate);
+            selectedDateElement.dataset.value = selectedDate;
+
+            populateDates();
+        });
+
+        daysElement.appendChild(dayElement);
+    }
+}
+
+// 종료일 - 월별 일자 생성
+function populateDatesEnd(event) {
+	daysElementEnd.innerHTML = "";
+	let amountDays = 31;
+
+    if (month == 1){
```

Comment:
조건문뒤나 함수뒤나 띄어쓰기 컨벤션을 동일하게 통일성을 신경써서 해주시면 좋을 거 같습니다.
아래와 같은 익스텐션도 있으니 참고해보세요
참고자료: https://ux.stories.pe.kr/150
```suggestion
    if (month == 1) {
```